web/client-api/src/main/java/io/deephaven/web/client/api/JsPartitionedTable.java

```diff
@@ -248,6 +259,26 @@ public int size() {
         return tables.size();
     }
 
+    /**
+     * An array of all the key columns used by the table
+     *
+     * @return Array of Column
+     */
+    @JsProperty
+    public Column[] getKeyColumns() {
+        return keyColumns;
+    }
+
+    /**
+     * An array of all the columns used by the table
```

Comment:
Might be clearer as "An array of the columns in the tables that can be retrieved from this partitioned table" or something to that effect?